pkg/store/postings_codec.go

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package store
+
+import (
+	"bytes"
+	encoding_binary "encoding/binary"
+	"fmt"
+
+	"github.com/golang/snappy"
+	"github.com/pkg/errors"
+	"github.com/prometheus/prometheus/tsdb/index"
+)
+
+const (
+	// these headers should not be prefix of each other
```

Comment:
Please make sure comments are full sentence (I think `make lint` should detect this)

Reply:
`make lint` failed on me on my mac :-( But I will check errors by CI.